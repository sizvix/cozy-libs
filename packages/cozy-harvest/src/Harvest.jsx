import React, { PureComponent } from 'react'
import { Form , Field } from 'react-final-form'
import GenForm from './GenForm'

export class Harvest extends PureComponent {
  render() {
      return <GenForm manifestFields={{'login':{type:'text'},'plop':{type:'password'} }} />
/*    return (
      <Form onSubmit={v=>console.log(v)} render={({ handleSubmit, form, submitting, pristine, values })=>(
        <div>
          <Field name="firstName" component="input" placeholder="First Name" />
          <Field name="interests" component={<input type="password" />} />
          <Field
            name="bio"
            render={({ input, meta }) => (
              <div>
                <label>Bio</label>
                <textarea {...input} />
                {meta.touched && meta.error && <span>{meta.error}</span>}
              </div>
            )}
          />
          <Field name="phone">
            {({ input, meta }) => (
              <div>
                <label>Phone</label>
                <input type="text" {...input} placeholder="Phone" />
                {meta.touched && meta.error && <span>{meta.error}</span>}
              </div>
            )}
          </Field>
          <pre>{JSON.stringify(values, 0, 2)}</pre>
        </div>
      )} />
    )	*/
  }
}

export default Harvest

