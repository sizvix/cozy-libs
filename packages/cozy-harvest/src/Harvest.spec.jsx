import React from 'react'
import Harvest from './Harvest'
import { shallow, configure } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'

configure({ adapter: new Adapter() })

describe('Harvest', () => {
  it('should render a div', () => {
    const component = shallow(<Harvest />).getElement()
    expect(component).toMatchSnapshot()
  })
})
