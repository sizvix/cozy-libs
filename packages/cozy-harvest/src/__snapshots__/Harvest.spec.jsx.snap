// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Harvest should render a div 1`] = `
<div>
  Initial implementation
</div>
`;
